src/memory-analyzer/gdb_api.h

```diff
@@ -24,14 +27,13 @@ Author: Malte Mues <mail.mues@gmail.com>
     defined(__MACH__)
 // clang-format on
 
-#ifndef CPROVER_MEMORY_ANALYZER_GDB_API_H
-#define CPROVER_MEMORY_ANALYZER_GDB_API_H
-#include <unistd.h>
+#  include <unistd.h>
 
-#include <exception>
-#include <forward_list>
+#  include <exception>
+#  include <forward_list>
+#  include <map>
 
-#include <util/exception_utils.h>
+#  include <util/exception_utils.h>
```

Comment:
The indenting of includes seems inconsistent with the rest of the code.

Reply:
This is the preprocessor indentation used as of #4461 (which only applies to preprocessor directives).

Reply:
That one seems highly unusual -- can clang-format be made not to do that?

Reply:
In my opinion, this one is entirely appropriate, because of the `#if` that precedes it. But maybe I'm the only one thinking so and if that's the case we can of course avoid this by adding a `clang-format off`.

Reply:
How much of a diff would it produce if we indent vs. if we don't.  I think that changes to the coding style that are a departure from the existing style (regardless of whether they are "right" or "wrong") need some thought and justification.  To me 'consistent' is a key priority.